benchmarks/BM_efficientnet/scripts/concatenate_results.py

```diff
@@ -0,0 +1,47 @@
+# The MIT License (MIT)
```

Comment:
I see that in the concatenated output has misaligned column description.
I would also add an info regarding batch size when concatenating.

Reply:
This behaviour happens due to a bug in `perf_analyzer`. I'm waiting for the response from Triton team to learn what will be the action here.

Reply:
And this is a good idea about the batch size. Let's wait until the response, so that we know whether we need to workaround the first issue. This would impact the implementation of adding the batch size info.

Reply:
https://github.com/triton-inference-server/client/pull/385